acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/transaction/privacy/PrivacyRequestFactory.java

```diff
@@ -51,4 +82,46 @@ public String getTransactionKey() {
       return getResult();
     }
   }
+
```

Comment:
Can the above response classes be moved to sit with your response classes at the top of the class